server/singleprocess: `TestService_waitOnJobStarted` test flake failure

**Describe the bug**

`TestService_waitOnJobStarted - github.com/hashicorp/waypoint/internal/server/singleprocess`

```
Failed
=== RUN   TestService_waitOnJobStarted
2021-06-01T18:29:43.394Z [INFO]  poll_queuer: starting
2021-06-01T18:29:43.394Z [INFO]  prune: starting
2021-06-01T18:29:43.394Z [INFO]  http: HTTP listener not specified, HTTP API is disabled
2021-06-01T18:29:43.394Z [INFO]  grpc: starting gRPC server: addr=127.0.0.1:45721
2021-06-01T18:29:43.395Z [INFO]  grpc: /hashicorp.waypoint.Waypoint/UpsertDeployment request
2021-06-01T18:29:43.411Z [INFO]  grpc: /hashicorp.waypoint.Waypoint/UpsertDeployment response: error=<nil> duration=16.836448ms
    service_exec_test.go:496: 
        	Error Trace:	service_exec_test.go:496
        	Error:      	Max difference between 1 and 1.386599527 allowed is 0.1, but difference was -0.386599527
        	Test:       	TestService_waitOnJobStarted
2021-06-01T18:29:44.799Z [INFO]  grpc: shutting down gRPC server
--- FAIL: TestService_waitOnJobStarted (1.60s)
```

**Steps to Reproduce**

The test sometimes fails on CircleCI. It is not 100% of the time...

**Expected behavior**

The test should pass

--- a/internal/server/singleprocess/service_exec_test.go
+++ b/internal/server/singleprocess/service_exec_test.go
@@ -493,7 +493,7 @@ func TestService_waitOnJobStarted(t *testing.T) {
 	require.NoError(err)
 	dur := time.Since(ts)
 
-	require.InDelta(1.0, dur.Seconds(), 0.1)
+	require.InDelta(1.0, dur.Seconds(), 1.0)
 
 	require.Equal(pb.Job_ERROR, js)
 }

